### ISSUE TITLE
firmware stubber v1.5.3 generates duplicate __init__ for classes

### ISSUE THREAD

**Issue by Josverl** [Josverl/micropython-stubber#164]
**Describe the bug**

Each class will have 2 inits

```
    def __init__(self, *argv, **kwargs) -> None:
        """"""
        ...

    def __init__(self, *args) -> None:
        ...

```

**Comment by Josverl** [comment]
related issue #150

**Comment by Josverl** [comment]
fixed by : createsubs v1.5.4 Better exeptions fix missed updates to mem and stub variants : 
e1cd179757156f8d978eef27bf3d69d4c14b0971
